src/components/titlePanel/createTitlePanel.ts

```diff
@@ -0,0 +1,90 @@
+import { VNodeProperties } from '@dojo/interfaces/vdom';
```

Comment:
I'll add specific comments, but I'm working off the a11y specs in https://www.w3.org/TR/wai-aria-practices-1.1/#accordion